src/routes/embedded/account/backup-wallet/backup-wallet-qrcode.tsx

```diff
@@ -0,0 +1,102 @@
+import type { JWKInterface } from "@dha-team/arbundles";
+import { useState, useCallback, useEffect } from "react";
+import { toast } from "react-toastify";
+import { Flex } from "~components/common/Flex";
+import { CopyToClipboard } from "~components/CopyToClipboard";
+import { Text } from "~components/embed/ui";
+import { OnboardingCard } from "~components/embed/ui/molecules/card/onboarding-card/OnboardingCard";
+import { useEmbedded } from "~utils/embedded/embedded.hooks";
+import { useLocation } from "~wallets/router/router.utils";
+import type { LocalWallet } from "~wallets/wallets.types";
+import browser from "webextension-polyfill";
+import { dataToFrames } from "qrloop";
+import { freeDecryptedWallet } from "~wallets/encryption";
+import { QRCodeWrapper } from "~components/QRCodeWrapper";
+import { QRCodeLoop } from "~components/QRCodeLoop";
+import { useTheme } from "~components/embed/contexts/ThemeContext";
+
+export function AccountBackupWalletQrCodeEmbeddedView() {
+  const { isDarkMode } = useTheme();
+  const { navigate } = useLocation();
+  const { getDecryptedWallet } = useEmbedded();
+  const [isLoading, setIsLoading] = useState(false);
+  const [copied, setCopied] = useState(false);
+  const [frames, setFrames] = useState<string[]>([]);
+  const [decryptedWallet, setDecryptedWallet] = useState<LocalWallet<JWKInterface> | null>(null);
+
+  const fetchDecryptedWallet = useCallback(async () => {
+    try {
+      if (decryptedWallet) return;
+      setIsLoading(true);
+      const wallet = await getDecryptedWallet();
```

Comment:
Using a seedphrase instead of a keyfile (when available) would mean this works with a single QR, but not sure if that's good or bad (easier to peak over someone's shoulder and steal it). 

Maybe just adding the seedphrase inside the keyfiles would be sufficient in order to allow users to preserve access to them.